src/test/AssetBasedIntentCurveLibHarness.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {
+    AssetBasedIntentCurve,
+    evaluate,
+    validate,
+    parseAssetType,
+    parseCurveType,
+    parseEvaluationType,
+    isRelativeEvaluation,
+    CurveType,
+    EvaluationType
+} from "../standards/assetbased/AssetBasedIntentCurve.sol";
+import {AssetType} from "../standards/assetbased/utils/AssetWrapper.sol";
+
+library AssetBasedIntentCurveLibHarness {
```

Comment:
External functions can receive calldata parameters and `AssetBasedIntentCurveLib` is not an existing external library anymore. For tests, this library is required to test the now free functions of ex-`AssetBasedIntentCurveLib` outside a `handleIntents` call, which itself is an external function and can receive a calldata parameter.